app/javascript/redux/greetings/greetingSlice.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
```

Comment:
- [ ] App doesn't compile successfully due to missing modules in the package.json file. Let's kindly double-check app and make sure it is working properly :+1:
![Screenshot from 2023-12-01 00-45-22](https://github.com/noelfoka/hello-rails-react-one/assets/63087491/66e29acd-c929-47bf-977c-b36b32f600d6)